src/System.Runtime.Extensions/tests/Performance/Perf.ArrayList.cs

```diff
@@ -0,0 +1,454 @@
+                    }                    
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = 10000)]
+        public void RemoveRange()
+        {
+            var random = new Random();
+            int size = (int)Benchmark.InnerIterationCount;
+            ArrayList elements = new ArrayList(size);
+            for (int index = 0; index < size; index++)
+            {
+                elements.Add(random.Next());
+            }
+
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                ArrayList[] elementsCollection = new ArrayList[size];
+                for (int index = 0; index < size; index++)
+                {
+                    elementsCollection[index] = new ArrayList(elements);
+                }
+
+                using (iteration.StartMeasurement())
+                {
+                    for (int index = 0; index < Benchmark.InnerIterationCount; index++)
+                    {
+                        elementsCollection[index].RemoveRange(1, size - 2);
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = 1000)]
+        public void Reverse()
+        {
+            int startIndex = 0;
+            int count = 100000;
+            ArrayList elements = CreateArrayListOfInts(count);
+
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
+                        elements.Reverse(startIndex, count);
+                    }                   
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = 10000000)]
+        public void SetRange()
+        {
+            int startIndex = 0;
+            int size = 100000;
+            ArrayList elements = CreateArrayListOfInts(size);
+            var newElemnts = new ArrayList(size);
+
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
+                        elements.SetRange(startIndex, elements);
+                    }                    
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = 20000000)]
+        public void GetRange()
+        {
+            int startIndex = 0;
+            int size = 100000;
+            ArrayList elements = CreateArrayListOfInts(size);
+
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
+                        var newElements = elements.GetRange(startIndex, size);
```

Comment:
Nit: the `var newElements` isn't necessary

Reply:
Fixed.